packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -63,6 +65,7 @@ module.exports = {
         },
         {
             name: 'serve',
+            alias: 's',
             scope: 'external',
```

Comment:
Alias `s` is already occupied by `sourcemap`, should be removed from there. 
https://github.com/webpack/webpack-cli/blob/ac2e52c2ef8aa74aa795476be94ce7c968b8d1c5/packages/webpack-cli/lib/utils/cli-flags.js#L168-L175

Reply:
`sourcemap` is a `cli-flag` as oppoesd to `serve` which is a command.
`webpack s` vs `webpack -s <value>`

Reply:
oh yes, my bad :-)